### ISSUE TITLE
1.0.0-pre.1 - Android is the only supported platform for now

### ISSUE THREAD

**Issue by ad-on-is** [bdlukaa/native_admob_flutter#20]
I'm using `1.0.0-pre.1` and I keep getting this error when trying to run on an iPhone. Even that I intend to use BannerAds on iOS for now.

```
[VERBOSE-2:ui_dart_state.cc(186)] Unhandled Exception: 'package:native_admob_flutter/src/utils.dart': Failed assertion: line 23 pos 10: 'Platform.isAndroid': Android is the only supported platform for now
#0      _AssertionError._doThrowNew (dart:core-patch/errors_patch.dart:46:39)
#1      _AssertionError._throwNew (dart:core-patch/errors_patch.dart:36:5)
#2      assertPlatformIsSupported (package:native_admob_flutter/src/utils.dart:23:10)
#3      MobileAds.initialize (package:native_admob_flutter/src/mobile_ads.dart:113:5)
#4      main (package:blitzcoin/main.dart:31:19)
#5      _runMainZoned.<anonymous closure>.<anonymous closure> (dart:ui/hooks.dart:140:25)
#6      _rootRun (dart:async/zone.dart:1354:13)
#7      _CustomZone.run (dart:async/zone.dart:1258:19)
#8      _runZoned (dart:async/zone.dart:1788:10)
#9      runZonedGuarded (dart:async/zone.dart:1776:12)
#10     _runMainZoned.<anonymous closure> (dart:ui/hooks.dart:133:5)
#11     _delayEntrypointInvocation.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:283:19)
#12     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:184:12)
```

**Comment by bdlukaa** [comment]
Ooh sorry I forgot about that! Will be fixed in the next version

**Comment by ad-on-is** [comment]
Just so that I can test RN, can I remove this line?
`assert(Platform.isAndroid, 'Android is the only supported platform for now');` in `assertPlatformIsSupported()`

**Comment by bdlukaa** [comment]
yes

**Comment by bdlukaa** [comment]
Fixed in [1.0.0](https://pub.dev/packages/native_admob_flutter)
